src/lib/layout/index.spec.ts

```diff
@@ -16,7 +15,62 @@ rootLogger.settings({
   },
 })
 
-const ctx = createTestContext()
+/**
+ * Helpers
+ */
+
+// In-memory file tree
+type VirtualFS = {
+  [path: string]: string | VirtualFS
+}
+
+const layoutContext = TestContext.create((opts: { tmpDir: () => string }) => {
```

Comment:
Feels like this should be typed via type reference to the contributor. This setups a clear dependency relationship.